src/main/java/org/broadinstitute/hellbender/tools/exome/allelefraction/MinorAlleleFractionCache.java

```diff
@@ -60,14 +48,14 @@ private static double estimateMinorAlleleFraction(final int a, final int r, fina
             final double refMinorLogLikelihood =
                     r * (logf + logBias) + a * logOneMinusf - (a + r) * logRefMinorDenominatorTerm;
             return GATKProtectedMathUtils.logSumExp(altMinorLogLikelihood, refMinorLogLikelihood);
-        });
+        };
 
-        return OPTIMIZER.optimize(objective, GoalType.MAXIMIZE, searchInterval, BRENT_MAX_EVAL).getPoint();
+        return OptimizationUtils.argmax(objective, 0.0, 0.5, initialEstimate);
```

Comment:
Could extract 0.5 here and throughout (to be consistent with what was done above in `AlleleFractionSamplers`), but I don't have a strong opinion.